programs/witness_node/main.cpp

```diff
@@ -96,6 +100,9 @@ int main(int argc, char** argv) {
       auto es_objects_plug = node->register_plugin<es_objects::es_objects_plugin>();
       auto grouped_orders_plug = node->register_plugin<grouped_orders::grouped_orders_plugin>();
       auto api_helper_indexes_plug = node->register_plugin<api_helper_indexes::api_helper_indexes>();
+      #ifdef QUERY_TXID_PLUGIN_ABLE
```

Comment:
Please let precompiler directives always start at the first column, don't indent them.